src/global_logic.js

```diff
@@ -446,6 +449,9 @@ module.exports.calcBasedOneSummon = function (summonind, prof, buff, totals) {
         if (key == "Djeeta") {
             hpCoeff += 0.01 * totals["Djeeta"]["job"].shugoBonus;
         }
+        if (totals[key]["EXLB"]["WED"]){
+            hpCoeff += 0.01 * 10;
```

Comment:
can be `hpCoeff += 0.10;` like other lines. 0.10 for 10% instead of 0.1

Reply:
Updated on latest branch